optaweb-employee-rostering-frontend/pom.xml

```diff
@@ -34,7 +34,6 @@
     <node.version>v11.6.0</node.version>
     <npm.version>6.9.0</npm.version>
     <sonar.sources>src</sonar.sources>
-    <sonar.test.inclusions>**/*test.ts,**/*test.tsx</sonar.test.inclusions>
```

Comment:
And we can't simply use `optaweb-employee-rostering-frontend/**/*test.ts` here because that doesn't work - results in counting tests as sources and measuring their coverage.